demo_scripts/data_providers/quandl_data_provider_demo.py

```diff
@@ -55,21 +55,6 @@ def main():
     data = data_provider.get_history(tickers=ticker, start_date=start_date, end_date=end_date)
     print(data)
 
-    print('Futures - ICE')
-    ticker = QuandlTicker('OZ2019', 'ICE')
-    data = data_provider.get_history(tickers=ticker, start_date=start_date, end_date=end_date)
-    print(data)
-
-    print('Futures - CME')
-    ticker = QuandlTicker('EMF2018', 'CME')
-    data = data_provider.get_history(tickers=ticker, start_date=start_date, end_date=end_date)
-    print(data)
-
-    print('Futures - EUREX')
-    ticker = QuandlTicker('FCXEZ2018', 'EUREX')
-    data = data_provider.get_history(tickers=ticker, start_date=start_date, end_date=end_date)
-    print(data)
-
```

Comment:
Thanks for testing the demos! 

The style_check tests are failing due to these issues:
"
./demo_scripts/data_providers/quandl_data_provider_demo.py:31:41: E261 at least two spaces before inline comment
./demo_scripts/data_providers/quandl_data_provider_demo.py:31:[8](https://github.com/quarkfin/qf-lib/actions/runs/7180218391/job/19558570359?pr=142#step:5:9)7: W291 trailing whitespace
"

Could you please fix those? If you want to run these checks locally you can just run "flake8 ."